src/openms_gui/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@
 project("OpenMS_GUI")
 cmake_minimum_required(VERSION 3.15 FATAL_ERROR)
 
-set(CMAKE_AUTOUIC OFF) # leave off since this is broken, see qt5_wrap_ui below
+set(CMAKE_AUTOUIC ON) # convert .ui files to ui_....h files using CMake + Qt's UIC
```

Comment:
think I had this is in the past but did not work

Reply:
I thought this is fixed within the 10+ years that we rolled our own solution, especially with Qt6.
It works fine on Windows, but that's not helping if it breaks elsewhere...

Reply:
fixed now. The problem was, that some GUI tests require the UIC'd headers, so they need to be part of the interface of OpenMS_GUI. CMake does not do that by default, since the headers are considered an implementation detail.